packages/esm-ward-app/src/ward-patient-card/row-elements/ward-patient-obs.tsx

```diff
@@ -0,0 +1,73 @@
+import { SkeletonText, Tag } from '@carbon/react';
+import { type OpenmrsResource, translateFrom } from '@openmrs/esm-framework';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { type PatientObsElementConfig } from '../../config-schema';
+import { useObs } from '../../hooks/useObs';
+import { type WardPatientCardElement } from '../../types';
+import styles from '../ward-patient-card.scss';
+import { moduleName } from '../../constant';
+
+// prettier-ignore
+const obsCustomRepresentation = 
+  'custom:(uuid,display,obsDatetime,value,' + 
+    'concept:(uuid,display),' + 
+    'encounter:(uuid,display,' + 
+      'visit:(uuid,display)))';
+
+const wardPatientObs = (config: PatientObsElementConfig) => {
+  const WardPatientObs: WardPatientCardElement = ({ patient, visit }) => {
+    const {
+      conceptUuid,
+      onlyWithinCurrentVisit,
+      orderBy,
+      limit,
+      label,
+      labelI18nModule: labelModule,
+      displayType,
+    } = config;
+    const { data, isLoading } = useObs({ patient: patient.uuid }, obsCustomRepresentation);
```

Comment:
This overall looks great, but this in particular is not going to perform, assuming I am understanding that this gets _all_ obs for the patient, and then filters down to those of interest.  We will definitely need a more efficient endpoint that does much of this at the DB +/- server level.

Reply:
Shouldn't be a concern after I change `useObs` to also take in the concept as Mark suggested.